src/styles/main.scss

```diff
@@ -1,3 +1,181 @@
+  &__list {
+    display: flex;
+    list-style: none;
+    text-transform: uppercase;
+  }
+
+  &__item {
+    color: $secondary;
+    position: relative;
+    padding: 22.5px 0 23.5px;
+    cursor: pointer;
+    font-family: "Roboto", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 14px;
+    &:not(:last-child) {
+      margin-right: 20px;
+    }
+    &:hover {
+      color: $primary;
+    }
+  }
+  .is-active {
+    color: $primary;
+    &::after {
+      content: "";
+      position: absolute;
+      bottom: 0;
+      left: 0;
+      right: 0;
+      background: $primary;
+      height: 4px;
+      border-radius: 8px;
+    }
+  }
+}
+
+.card {
+  padding: 32px 16px 16px;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin-bottom: 4px;
+    color: $secondary;
+  }
+  &__description,
+  &__reviews {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+  }
+  &__description {
+    color: $tertiary;
+    margin-bottom: 16px;
+  }
+  &__rating {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-bottom: 24px;
+    .stars {
+      display: flex;
+      &__star {
+        width: 16px;
+        height: 16px;
+        background: url("../images/star.svg") center no-repeat;
+        &:not(:last-child) {
+          margin-right: 4px;
+        }
+      }
+      &--4 {
+        .stars__star:nth-child(-n+4) {
+          background: url("../images/star-active.svg") center no-repeat;
+        }
+      }
+    }
+  }
+  &__reviews {
+    color: $secondary;
+  }
+  &__price {
+    &-wraper {
+      display: flex;
+      align-items: center;
+      justify-content: space-between;
+      margin-bottom: 16px;
+    }
+    &-text {
+      font-size: 12px;
+      line-height: 18px;
+      color: $tertiary;
+      font-weight: 400;
+    }
+    &-amount {
+      font-size: 16px;
+      line-height: 18px;
+      font-weight: 700;
+      color: $secondary;
+    }
+  }
+  &__btn {
+    &:link,
+    &:visited {
+      font-weight: 700;
+      font-size: 14px;
+      line-height: 16px;
+      text-transform: uppercase;
+      text-decoration: none;
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      height: 40px;
+      background-color: $primary;
+      border-radius: 5px;
+      border: 1px solid $primary;
+      color: #fff;
+    }
+
+    &:hover,
+    &:active {
+      background-color: #fff;
+      color: $primary;
+    }
+  }
+  &:not(:nth-child(4n)) {
+    margin-right: 48px;
+  }
+  &:not(:nth-last-child(-n + 4)) {
+    margin-bottom: 46px;
+  }
+}
+
+.cards {
+  width: 1024px;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  padding: 50px 40px;
+  margin: 0 auto;
```

Comment:
please check that you have taken into account all the items on the [checklist](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md), and from this [list](https://mate-academy.github.io/fe-program/css/typical-bem-mistakes-en) also